docs/utilities/gestures.md

```diff
@@ -137,7 +137,9 @@ const gesture = createGesture({
 
 ## Basic Gestures
 
-### Usage
+import Basic from '@site/static/usage/v7/gestures/basic/index.md';
+
+<Basic />
 
 ````mdx-code-block
```

Comment:
Sorry to kool-aid man myself into your PR (i'm working on a gesture playground and was checking to see if we used the same directory structure), but I think you can remove the old code sample tabs here.

Reply:
Always happy for additional 👀  you are so correct, I didn't ever scroll to see that 🤣